### DISCUSSION TITLE
Include rdb kernel module for Rock 5B

### DISCUSSION THREAD

**Discussion by skyuplam** [MichaIng/DietPi#6480] [Q&A] [answered]
It seems that the rdb kernel module for Rock 5B build. Is it possible to include that for the next release? It is required to run Rook/Ceph (https://rook.io/docs/rook/v1.11/Getting-Started/Prerequisites/prerequisites/#rbd), similar to this request https://github.com/MichaIng/DietPi/discussions/6402

**Comment by Joulinar** [comment]
We don't build own kernel for this device.

**Reply by skyuplam** [reply]
Thanks for your promptly reply and where should I ask for? Armbian?

**Reply by Joulinar** [reply]
@MichaIng which kernel we use on Rock 5B?

**Reply by MichaIng** [reply, marked as answer]
Jep we use Armbian's "legacy" kernel builds (Linux 5.10 based on vendor sources). You can of course ask Armbian, but test first with their image(s): https://www.armbian.com/rock-5b/

And do not mentioned DietPi at all when posting in their forums, just do as Armbian user using an Armbian image. They do not like to do work for DietPi users.
